invertedai_drive/utils.py

```diff
@@ -167,3 +258,42 @@ def _make_box_layout(self):
             margin="0px 10px 10px 0px",
             padding="5px 5px 5px 5px",
         )
+
+
+class IAI_Logger(logging.Logger):
+    def __init__(
+        self,
+        name: str = "IAI_Logger",
+        level: str = "WARNING",
+        consoel: bool = True,
+        log_file: bool = False,
+    ) -> None:
+
+        log_level = level if type(level := logging.getLevelName(level)) == int else 40
```

Comment:
Since default level is WARNING I assume it makes more sense to fallback to 30 ? unless you think it should all default to ERROR